.ci/jenkins_job_build.sh

```diff
@@ -174,6 +174,10 @@ esac
 "${ci_dir_name}/setup.sh"
 
 if [ "${METRICS_CI}" == "false" ]; then
+	if [ "${kata_repo}" != "${tests_repo}" ]; then
+		make test
+	fi
+
```

Comment:
Was code coverage working on any of the repos without this?

Reply:
It was working with 1.x contributions in runtime and probably other repos, but it wasn't working with the kata-containers/kata-containers repo (2.0-dev branch) because this section was missing in the 2.0-dev branch (maybe @chavafg can correct me if I'm wrong)

Reply:
yeah, this is working on all 1.x branches (including master) on all repos that have golang code.

Reply:
@cmaf @chavafg Can you explain how this fixes coverage for 2.x?

Reply:
iirc, the `make test` target creates a report, which then codecov uses for reporting coverage. 
/cc @jodh-intel

Reply:
Yes, `make test` calls https://github.com/kata-containers/tests/blob/master/.ci/go-test.sh eventually, which creates the magic [`coverage.txt`](https://github.com/kata-containers/tests/blob/master/.ci/go-test.sh#L42) file.

However, this is golang only - we need to get this working for rust too.